lib/auth.js

```diff
@@ -16,6 +16,7 @@ exports.AUTH_STRATEGY = 'dogfood';
 exports.AUTH_SCHEME = 'bearer';
 
 exports.SCOPE_CLIENT_MANAGEMENT = 'oauth';
+exports.SCOPE_ACCOUNT_MANAGEMENT = 'account';
```

Comment:
How about, since these scopes take up a global namespace of sorts, it be a sub-scope of `oauth`? So, to get tokens for a user, you'd need the scope `oauth:tokens` or something. The console, with its `oauth` scope, gets the subscope for free, and we leave "account" out of this. Thoughts @kitcambridge @rfk?